db/migrate/20200518121043_create_todo_items.rb

```diff
@@ -1,7 +1,7 @@
 class CreateTodoItems < ActiveRecord::Migration[6.0]
   def change
     create_table :todo_items do |t|
-      t.string :content
+      t.string :content, default: "Title-1"
```

Comment:
Is it a good idea to give a default value to content?

Reply:
I was trying to display Validation Errors in Views but it shows  - undefined method `errors' for nil:NilClass.

snapshot of the _form.html.erb
<img width="905" alt="Screenshot 2020-05-19 at 1 38 02 PM" src="https://user-images.githubusercontent.com/25845059/82301491-f8d4c980-99d5-11ea-9071-0c958563244c.png">

todo_items_controller

![image](https://user-images.githubusercontent.com/25845059/82301601-17d35b80-99d6-11ea-9da9-1d51a1c3af4c.png)


Can you help me with this?